agent/requirements/test.txt

```diff
@@ -9,268 +9,202 @@
 asynctest==0.13.0 \
     --hash=sha256:5da6118a7e6d6b54d83a8f7197769d046922a44d2a99c21382f0a6e4fadae676 \
     --hash=sha256:c27862842d15d83e6a34eb0b2866c323880eb3a75e4485b079ea11748fd77fac
-# via -r requirements/test.in
-# via
-#   aiohttp
-#   pytest
-black==21.10b0 \
-    --hash=sha256:6eb7448da9143ee65b856a5f3676b7dda98ad9abe0f87fce8c59291f15e82a5b \
-    --hash=sha256:a9952229092e325fe5f3dae56d81f639b23f7131eb840781947e4b2886030f33
-# via -r requirements/test.in
+    # via -r requirements/test.in
+black==21.12b0 \
```

Comment:
odd that we are using b0 versions of black, not a formal release.

Reply:
https://github.com/psf/black/tags looks like all black versions are bX for some reason.